### PR TITLE
Enable limit_latest_versions to be configured from configuration

### PR DESCRIPTION
**Pull request by harshad16**
Updated the configuration of limit_latest_versions from the configuration file (.thoth.yaml).
Fixes: #111

### CODE REVIEW

File: .thoth.yaml [hunk 1]
```diff
@@ -9,3 +9,4 @@ runtime_environments:
       version: "7"
     python_version: "3.6"
     recommendation_type: latest
+    limit_latest_versions: None
```
Comment (fridex) [inline .thoth.yaml]: Could you put this configuration entry into configuration template present in `thamos/thamos/data/defaultThoth.yaml`? That's the configuration which gets expanded on `thamos config`.

Comment (harshad16) [reply .thoth.yaml]: surely, will do :+1:

Comment (fridex) [reply .thoth.yaml]: Ah, just spotted it now... In yaml, you need to set value to `null` to have "None" value (None is python specific):

```python
In [1]: import yaml

In [2]: yaml.safe_load("foo: None")
Out[2]: {'foo': 'None'}

In [3]: yaml.safe_load("foo: null")
Out[2]: {'foo': None}
```

Sorry, tired eyes.

Comment (harshad16) [reply .thoth.yaml]: Thanks for letting me know.
Stupid of me.
I' ll change the values.

**Pull request merged**